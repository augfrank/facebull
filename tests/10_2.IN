M0   C0   C2  2502
M1   C0   C6  2631
M2   C0   C7  2603
M3   C1   C4  2791
M4   C1   C8  2492
M5   C2   C3  2759
M6   C3   C0  2319
M7   C3   C6  2042
M8   C3   C7  2665
M9   C4   C3  2844
M10   C4   C5  2672
M11   C4   C6  2916
M12   C5   C0  2905
M13   C5   C1  2054
M14   C5   C8  2055
M15   C5   C9  2224
M16   C6   C2  2800
M17   C7   C0  2139
M18   C7   C1  2810
M19   C7   C3  2186
M20   C7   C8  2471
M21   C7   C9  2434
M22   C8   C2  2627
M23   C8   C9  2277
M24   C9   C4  2703
M25   C9   C6  2332
M26   C9   C8  2060
