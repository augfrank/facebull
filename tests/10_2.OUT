25418
2 3 5 6 10 14 16 18 23 25
